README.md

```diff
@@ -452,25 +447,25 @@ Capybara::Screenshot::Diff.area_size_limit = 42
 ### Skipping an area
 
 Sometimes you have expected change that you want to ignore.
-You can use the `skip_area` option to the `screenshot` method to ignore an area:
+You can use the `skip_area` option with `[left, top, right, bottom]` or css selector like `'body #skiped_element'' to the `screenshot` method to ignore an area:
 
 ```ruby
 test 'unstable area' do
   visit '/'
-  screenshot 'index', skip_area: [17, 6, 27, 16]
+  screenshot 'index', skip_area: [[17, 6, 27, 16], 'input.-to-skip', '.footer']
```

Comment:
Typo?  Should be 'input.to-skip' ?

Reply:
oh sorry this compressed BEM notation, will remove dash